.goreleaser.yml

```diff
@@ -0,0 +1,113 @@
+# .goreleaser.yaml
+project_name: cdk-data-availability
+
+release:
+  disable: false
+  draft: true
+  prerelease: auto
+
+builds:
+  - id: linux-amd64
+    main: ./cmd/
+    binary: cdk-data-availability
+    goos:
+      - linux
+    goarch:
+      - amd64
+    env:
+      - CGO_ENABLED=0
+    ldflags:
+      - -s -w
+      - -X github.com/0xPolygon/cdk-data-availability.Version={{ .Version }}
+      - -X github.com/0xPolygon/cdk-data-availability.GitRev={{ .Commit }}
+      - -X github.com/0xPolygon/cdk-data-availability.BuildDate={{ .Date }}
+      - -X github.com/0xPolygon/cdk-data-availability.GitBranch={{ .Branch }}
+
+  - id: linux-arm64
+    main: ./cmd/
+    binary: cdk-data-availability
+    goos:
+      - linux
+    goarch:
+      - arm64
+    env:
+      - CGO_ENABLED=0
+    ldflags:
+      - -s -w
+      - -X github.com/0xPolygon/cdk-data-availability.Version={{ .Version }}
+      - -X github.com/0xPolygon/cdk-data-availability.GitRev={{ .Commit }}
+      - -X github.com/0xPolygon/cdk-data-availability.BuildDate={{ .Date }}
+      - -X github.com/0xPolygon/cdk-data-availability.GitBranch={{ .Branch }}
+
+  - main: ./cmd/
```

Comment:
You can just assign an `id` to this and it will be picked up by the docker build for the corresponding arch